bodhi/client/__init__.py

```diff
@@ -625,14 +636,19 @@ def query_buildroot_overrides(url, user=None, mine=False, packages=None,
         expired (bool): If supplied, True returns only expired overrides, False only active.
         releases (unicode): If supplied, the overrides for these releases are queried.
         builds (unicode): If supplied, the overrides for these builds are queried.
+        rows (unicode): The limit of rows displayed per page for query result.
+                        Default: 20, Min: 1, Max: 100
+        page (unicode): If supplied, returns the results for a specific page number.
+                        Default: 1
```

Comment:
Let's not document these defaults here - they are API defaults and are not particular to the CLI.